test/Dialect/FIRRTL/const-prop-single-module.fir

```diff
@@ -0,0 +1,174 @@
+; RUN: circt-translate --import-firrtl %s | circt-opt --canonicalize | FileCheck %s
```

Comment:
Hey @fabianschuiki , I didn't notice this earlier, but please don't add unit tests that come from .fir files like this.  This will be fragile w.r.t. changes in the FIRParser (e.g. if it starts implicitly folding things, which is an entirely reasonable thing for it to do).  Could you switch this to a .mlir test instead?

Reply:
Sure thing, will do.